src/System.Diagnostics.Process/src/System/Diagnostics/Process.OSX.cs

```diff
@@ -9,14 +9,16 @@ namespace System.Diagnostics
 {
     public partial class Process
     {
+        private const int NanoSecondsTo100NanoSecondsFactor = 100;
+
         /// <summary>Gets the amount of time the process has spent running code inside the operating system core.</summary>
         public TimeSpan PrivilegedProcessorTime
         {
             get
             {
                 EnsureState(State.HaveNonExitedId);
                 Interop.libproc.rusage_info_v3 info = Interop.libproc.proc_pid_rusage(_processId);
-                return new TimeSpan(Convert.ToInt64(info.ri_system_time));
+                return new TimeSpan(Convert.ToInt64(info.ri_system_time) / NanoSecondsTo100NanoSecondsFactor);
```

Comment:
Nit: looking at it again, we should do the divide before passing the result to Convert.ToInt64:
```C#
return new TimeSpan(Convert.ToInt64(info.ri_system_time / NanoSecondsTo100NanoSecondsFactor));
```
Same for the other places.

Reply:
makes sense 👍